attachments_component/site/src/Controller/AttachmentsController.php

```diff
@@ -89,9 +90,12 @@ public function displayString($parent_id, $parent_type, $parent_entity,
 		$model = $this->getModel('Attachments', 'Site');
 		if ( !$model ) {
 			$errmsg = Text::_('ATTACH_ERROR_UNABLE_TO_FIND_MODEL') . ' (ERR 60)';
-			throw new \Exception($errmsg, 500);
+			//throw new \Exception($errmsg, 500);
+			return;
 			}
-
+		if ($attachmentsid) {
+			$parent_id = '?';
```

Comment:
Why do you change parent_id to ?

Reply:
the idea was to be able to select other attachments as current article